django_ilmoitin/api/schema/types.py

```diff
@@ -0,0 +1,50 @@
+import graphene
+from django.apps import apps
+from django.conf import settings
+from graphene_django import DjangoObjectType
+from parler.utils.context import switch_language
+
+from django_ilmoitin.models import NotificationTemplate
+from django_ilmoitin.utils import render_preview
+
+LanguageEnum = graphene.Enum(
+    "Language", [(lang[0].upper(), lang[0]) for lang in settings.LANGUAGES]
+)
+
+
+class NotificationTranslationType(DjangoObjectType):
+    language_code = LanguageEnum(required=True)
+    subject = graphene.String()
+    body_html = graphene.String()
+    body_text = graphene.String()
+    preview = graphene.String()
+
+    class Meta:
+        model = apps.get_model("django_ilmoitin", "NotificationTemplateTranslation")
+        use_connection = True
```

Comment:
why do you set this to `True`? 🤔

Reply:
Apparently it's not needed anymore 🤔 It was causing some error when trying to query saying that there was no connection, but apparently now that `NotificationTemplate` implements the `relay.Node` interface, that's solved.

Reply:
ah, alright 👍